frontend/tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "target": "es2017",
     "module": "es2020",
     "lib": [
-      "es2018",
+      "es2022",
```

Comment:
Why this upgrade? Do we really need it? Does it have impact on the generated frontend code (can it work on last years browsers)?

Reply:
The upgrade to ES2022 is required because `Array.at(index)` is used in the create-model component, which was first introduced in ES2022, and the test runner complains about it. However, I only checked if we can use ES2022 with our current node version and didn't think about possible problems with the browser version. As far as I can see (https://caniuse.com/mdn-javascript_builtins_array_at), most browsers support using Array.at since mid-2021 (except Safari, which supports it since March this year). So I think we should either just add "es2022.array" in addition to "es2018" so that we don't include all ES2022 functions, or just rewrite the "Array.at" part